### PR TITLE
[Resto Shaman] Conduits

### PR DESCRIPTION
**Pull request by niseko**
image

### CODE REVIEW

File: src/parser/shaman/shared/talents/EarthShield.tsx [hunk 1]
```diff
@@ -18,8 +18,9 @@ import Combatants from 'parser/shared/modules/Combatants';
 import { Trans } from '@lingui/macro';
 
 import { HEALING_ABILITIES_AMPED_BY_EARTH_SHIELD } from 'parser/shaman/shared/constants';
+import { EMBRACE_OF_EARTH_RANKS } from 'parser/shaman/restoration/constants';
 
-const EARTHSHIELD_HEALING_INCREASE = 0.20; // TODO add conduit
+export const EARTHSHIELD_HEALING_INCREASE = 0.20;
 
 class EarthShield extends Analyzer {
```
Comment (abelito75) [inline src/parser/shaman/shared/talents/EarthShield.tsx:25]: This is a core spell now right? Shouldn't it go under that and not talents?

**Review by abelito75 — APPROVED** [review]

File: src/parser/shaman/shared/talents/EarthShield.tsx (hunk 1 above)
Comment (niseko) [reply src/parser/shaman/shared/talents/EarthShield.tsx:25]: its a talent for 2 out of 3 specs

Comment (abelito75) [reply src/parser/shaman/shared/talents/EarthShield.tsx:25]: thats stupid

Comment (niseko) [reply src/parser/shaman/shared/talents/EarthShield.tsx:25]: no u

**Pull request merged**